pyanaconda/ui/gui/spokes/network.py

```diff
@@ -1185,7 +1191,7 @@ def refresh(self, device_name):
             log.warnig("device for interface %s not found", device)
             return
 
-        cons = device.filter_connections(self._nm_client.get_connections())
+        cons = self._device_filter_connections(device, self._nm_client.get_connections())
```

Comment:
This method is not defined in the class, but nevermind, I'll modify the patch a bit and submit it for rhel-8 branch. We can then port it to master after review.

Reply:
right!